### PR TITLE
fix(card-static): cta link not working on mobile

### PR DESCRIPTION
**Pull request by NSDrowned**
### Related Ticket(s)

https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/6490

### Description

- Footer's `::after` element (not used in this variation) blocking anchor link. Removed pointer events.
- Added `position: relative` to parent so `::after` only occupies the CTA's area, not the whole screen.

https://user-images.githubusercontent.com/4054756/123836782-4ef98900-d8c7-11eb-889f-20e961ad54cf.MP4

### CODE REVIEW

File: packages/web-components/src/components/card/card.scss [hunk 1]
```diff
@@ -18,6 +18,14 @@
     color: $inverse-link;
   }
 
+  &[color-scheme='light'] {
```
Comment (annawen1) [inline packages/web-components/src/components/card/card.scss:21]: does the static card always have the `light` color-scheme? what happens if we attach the styles to 

```
:host(#{$dds-prefix}-card-footer),
:host(#{$dds-prefix}-card-cta-footer)
```

**Comment by jeffchew** [comment]
@NSDrowned is this off of the release branch? it looks like you are targeting `master`.

**Review by jeffchew — CHANGES_REQUESTED** [review]
@NSDrowned can you reopen the PR based off the release branch? It looks like this was pulled from master.

File: packages/web-components/src/components/card/card.scss (hunk 1 above)
Comment (NSDrowned) [reply packages/web-components/src/components/card/card.scss:21]: @annawen1, yes it's always on in static. Can't find another way to target static card. Tested it on other variations and they all work fine except for the light schem.

**Comment by NSDrowned** [comment]
@jeffchew whoops, you got it!

**Comment by kennylam** [comment]
@NSDrowned Link functionality working fine for me now. But for some reason Safari adds a `margin-left` to the icon.

Chrome on left, Safari on right.
image

**Pull request closed without merging**